src/Worker/RemoveContact.php

```diff
@@ -41,10 +41,10 @@ public static function execute($id) {
 		// Now we delete the contact and all depending tables
 		$condition = ['uid' => $contact['uid'], 'contact-id' => $id];
 		do {
-			$items = Item::select(['id'], $condition, ['limit' => 100]);
+			$items = Item::select(['id', 'guid'], $condition, ['limit' => 100]);
 			while ($item = Item::fetch($items)) {
 				Logger::notice('Delete removed contact item', ['id' => $item["id"]]);
-				DBA::delete('item', ['id' => $item['id']]);
+				DBA::delete('item', ['id' => $item['id'], 'guid' => $item['guid']]);
```

Comment:
Why have you added the `guid` here? The `id` is the primary key, so adding the `guid` doesn't bring any benefit.